ingest-app/src/cmr/ingest/api/subscriptions.clj

```diff
@@ -82,6 +82,25 @@
                              query-params)
         gran-refs (search-gran-refs-with-sub-params request-context search-params)]))
 
+(defn- check-subscription-limit
+  "Given the  configuration for subscription limit, this valdiates that the
+  user has less than the limit before we allow more subscriptions to be
+  ingested by that user."
+  [request-context subscription]
+  (let [metadata (-> (:metadata subscription) (json/decode true))
```

Comment:
is -> needed in this case?

Reply:
It's possible I did something else wrong in removing it, but it seemed to break when I tried without the thread.